### ISSUE TITLE
Update AKS regions to include Brazil South/Central India/Japan West/South Africa North

### ISSUE THREAD

**Issue by superseb** [rancher/rancher#21232]
**What kind of request is this (question/bug/enhancement/feature request):**
task

**Steps to reproduce (least amount of steps as possible):**
Add Brazil South/Central India/Japan West/South Africa North to AKS regions

**Comment by davidnuzik** [comment]
Version: master-head (7/1/19)

I validated all the new regions per the PR.
(below you'll see `displayName` = `name` I use the plain `name` to validate API/HTTP)
Brazil South = brazilsouth
Central India = centralindia
Japan West = japanwest
South Africa North = southafricanorth
![Screen Shot 2019-07-01 at 4 19 50 PM](https://user-images.githubusercontent.com/45179589/60472104-5e806180-9c1c-11e9-846f-08e051df4647.png)

For each I was able to validate that UI send correct location, for example when validate via aksVersions and aksVirtualNetworks azure call `region: "brazilsouth"` and for rancher cluster creation (azureKubernetesServiceConfig) `location: "brazilsouth"`, etc

Confirmed AKS cluster comes up for each.
Confirmed that our Region/Location supplied matches what I see in Azure web console.
